addons/analytic/views/analytic_line_views.xml

```diff
@@ -112,8 +113,9 @@
         <field name="model">account.analytic.line</field>
         <field name="arch" type="xml">
             <pivot string="Analytic Items" sample="1">
+                <field name="plan_id" type="row"/>
```

Comment:
Analytic Items > Pivot view is not grouped by plan, correct?

Reply:
Yep, we know, you can't really do much about that since you first land on the list view, then switch to pivot, but it keeps the "Analytic Account" group by from the list view (which it should IMO).